server/api/gameController.js

```diff
@@ -0,0 +1,14 @@
+const createRoom = (playerInfo) => {
+  const roomId = (Math.random() * 100000000000).toFixed(0);
+  const pacmanOne = playerInfo;
+  const pacmanTwo = {};
```

Comment:
Generalize it instead of having the game available just for 2 players.

Reply:
But we are restricting that only two players can join a room.